test/e2e/e2e.yml

```diff
@@ -0,0 +1,93 @@
+# A basic flag custom resource
```

Comment:
I think it may be worth adding a "top level" comment explaining what this is (testing the mutating injection webhook by proxying through an nginx, which is also the "injected" workload).

Reply:
Good shout, done